File: PRAK305-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
int main ()
{
    int detik, menit, jam, hari, sisadetikA, sisadetikB, detikX;
 
    printf ("Masukan detik : " );
    scanf ("%d", &detik);

    sisadetikA = detik % 86400;
    sisadetikB = detik % 3600;
    detikX     = detik % 60;
    menit      = sisadetikB / 60;
    jam        = sisadetikA / 3600;
    hari       = detik /86400;
    
    if (detik >=1 &&  detik <60){
        printf ("00:00:%.2d\n", detik);
    }else if ( detik >=60 && detik < 3600) {
        printf ("00:%.2d:%.2d\n", menit, detikX);
    }else if (detik >=3600 && detik <86400) {
        printf ("%.2d:%.2d:%.2d\n", jam, menit, detikX);
    }else {
        printf ("%d hari %.2d:%.2d:%.2d\n", hari, jam, menit, detikX);
    } 
    printf ("\n");
}

File: PRAK305-2210817120007-NursyahnaPuteri.py
detik = int (input("masukan detik : "))

sisadetikA = detik % 86400
sisadetikB = detik % 3600
detikX     = detik % 60
menit      = sisadetikB // 60
jam        = sisadetikA // 3600
hari       = detik //86400

if detik >=1 and detik < 60 :
    print ("00:00:%.2d" % (detik))
elif detik >=60 and detik < 3600 :
    print ("00:%.2d:%.2d"% (menit, detikX))
elif detik >=3600 and detik < 86400 :
    print ("%.2d:%.2d:%.2d"% (jam, menit, detikX))
else :
    print ("%d hari %.2d:%.2d:%.2d"% (hari, jam, menit, detikX))